docs/fast_data/configuration/realtime_updater/common.md

```diff
@@ -248,22 +248,22 @@ Example:
         "projectionChanges": {
             "sv_pointofsale": {
                 "strategy": "__automatic__",
-                "topic": "my-tenant.development.my-database.sv-pointofsale.sv-trigger",
+                "topic": "my-tenant.development.my-database.sv-pointofsale.projection-change",
             }
         }
     },
     "another-projection": {
         "projectionChanges": {
             "sv_customer": {
                 "strategy": "__fromFile__[myStrategy]",
-                "topic": "my-tenant.development.my-database.sv-customer.sv-trigger"
+                "topic": "my-tenant.development.my-database.sv-customer.projection-change"
             }
         }
     }
 }
 ```
 
-When a message about `registry-json` happens, the projection changes will be saved on MongoDB, and it will be sent to the Kafka topic `my-tenant.development.my-database.sv-pointofsale.sv-trigger` as well.
+When a message about `registry-json` happens, the projection changes will be saved on MongoDB, and it will be sent to the Kafka topic `my-tenant.development.my-database.sv-pointofsale.projection-change` as well.
```

Comment:
remember to put in the breaking change of the service changelog and then in the release notes that we changed the topic from sv-trigger to projection-change

Reply:
we actually don't have a naming convention for the kafka `projection-change`. We used to "mix" the concept of mongo projection change, kafka projection change and sv-trigger message. This change you highlighted is to make the separation between them clear

Reply:
ok thank you! I was mixing the concepts as well